### PR TITLE
Remove most of hardcoded USD

### PR DESCRIPTION
**Pull request by daisy1754**
# Description
Set proper currency code everywhere except `getRoundAmount` in Cart.php, which I'll refactor in next PR

Fixes: https://app.asana.com/0/941920570700290/1130539126613441/f

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [x] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: Helper/Cart.php [hunk 1]
```diff
@@ -1353,6 +1355,7 @@ public function getCartData($paymentOnly, $placeOrderPayload, $immutableQuote =
      * @param $amount
      * @return  int
      */
+    // TODO remote this method
     public function getRoundAmount($amount)
```
Comment (roopakv) [inline Helper/Cart.php]: where is this used? if this is not used can we not get rid of it now?

Comment (daisy1754) [reply Helper/Cart.php]: it's used in 10+ different location which is why I wanted to remove this in separate PR

Comment (roopakv) [reply Helper/Cart.php]: ahhh makes sense.

File: Model/Api/ShippingMethods.php [hunk 2]
```diff
@@ -44,6 +44,7 @@
 use Bolt\Boltpay\Exception\BoltException;
 use Bolt\Boltpay\Helper\Discount as DiscountHelper;
 use Magento\SalesRule\Model\RuleFactory;
+use Magento\Setup\Exception;
```
[the remark this answers is not in the document]
Comment (daisy1754) [reply Model/Api/ShippingMethods.php]: good catch this was unused. removed.

**Pull request merged**